var/spack/repos/builtin/packages/gene-expression-volume/package.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class GeneExpressionVolume(PythonPackage):
+    """BBP command line interface for the re-alignment of 2D images and 
+    the creation of 3D gene expression volumes. 
+    Used by the atlas building pipeline.
+    """
+    homepage = "https://bbpgitlab.epfl.ch/nse/gene-expression-volume"
+    git      = "git@bbpgitlab.epfl.ch:nse/gene-expression-volume.git"
+
+    version('0.1.0', tag='gene-expression-volume-v0.1.0')
+
+    depends_on('py-setuptools', type=('build', 'run'))
+    depends_on('py-atlalign@0.6.0', type=('build', 'run'))
+    depends_on('py-click@7.0:', type=('build', 'run'))
+    depends_on('py-nptyping@1.0.1:', type=('build', 'run'))
+    depends_on('py-numpy@1.15.0:', type=('build', 'run'))
+    depends_on('py-scikit-image@0.017.2:', type=('build', 'run'))
+    depends_on('py-types-pyaml@5.4.0:', type=('build', 'run'))
+    depends_on('py-voxcell@2.7.4', type=('run', 'build'))
+    depends_on('py-pyaml@5.3.1:', type=('build', 'run'))
+    depends_on('py-pytest', type='test')
+    depends_on('py-mock', type='test')
+
```

Comment:
```suggestion

    def patch(self):
        filter_file('"tensorflow==[0-9.]+",', '', 'setup.py')
```

Reply:
@matz-e Thanks for looking into this. I have tried your suggestion, but a "new" problem arises then:

```
 File "/gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/deploy/libraries/2021-01-06/linux-rhel7-x86_64/gcc-9.3.0/py-setuptools-50.3.2-afe2g4/lib/python3.8/site-packages/pkg_resources/__init__.py", line 770, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'lpips_tf@ git+http://github.com/alexlee-gk/lpips-tensorflow.git#egg=lpips_tf' distribution was not found and is required by gene-expression-volume
``` 

I thought that the following could help:

```
def patch(self):
    filter_file('"lpips_tf.+=lpips_tf",', '', 'setup.py')
    filter_file('"tensorflow==[0-9.]+",', '', 'setup.py')
```
but then I get

```
  File "/gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/deploy/libraries/2021-01-06/linux-rhel7-x86_64/gcc-9.3.0/py-setuptools-50.3.2-afe2g4/lib/python3.8/site-packages/pkg_resources/__init__.py", line 770, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'tensorflow>=2.4.0' distribution was not found and is required by atlalign
```

which bewildered me.